XRTK.Oculus/Packages/com.xrtk.oculus/Runtime/Providers/Controllers/BaseOculusController.cs

```diff
@@ -28,6 +28,13 @@ protected BaseOculusController(IMixedRealityControllerDataProvider controllerDat
             NodeType = nodeType;
         }
 
+        /// <summary>
+        /// Local offset from the controller position defining where the grip pose is.
+        /// The grip pose may be used to attach things to the controller when grabbing objects.
+        /// </summary>
+        private readonly MixedRealityPose gripPoseOffset = new
```

Comment:
Can we move this variable to the base controller definition? Would seem to make sense to make this a generic thing as it is likely to be needed on multiple platforms.

Reply:
I thought about that. But essentially I don't think every controller has a grip pose, right?

Reply:
E.g. for the Oculus Remote it does not make much sense?

Reply:
I think we only had a pointer pose offset, but I suppose a grip pose offset would be useful in the base controller definition